Create a standardized form testing dataset

arinkulshi-skylight opened the pull request:
## Description
https://github.com/CDCgov/ReportVision/issues/254



Here is the actual dataset.
https://drive.google.com/drive/folders/1WS2FYn0BTxWv0juh7lblzdMaFlI7zbDd
You can download the data from google drive and drag and drop the two folders images and ground-truth. the script above will also pull the data from the individual datasets on HuggingFace

## Screenshots (if applicable)

## Related Issues
[Link any related issues or tasks from your project management system.]

## Checklist
- [x] The title of this PR is descriptive and concise.
- [x] My changes follow the style guidelines of this project.
- [x] I have added or updated test cases to cover my changes.
- [x] I've let the team know about this PR by linking it in the review channel

schreiaj reviewed (changes requested):
Please add a section in the readme about running this (and maybe interpreting results?) 

Additionally, do we need to update dependencies to include `datasets` I don't seem to have it installed inside my poetry shell.

arinkulshi-skylight commented on OCR/ocr/reportvision-dataset-1/medical_report_import.py:
```diff
@@ -0,0 +1,57 @@
+for split in dataset.keys():
+    split_data = dataset[split]
+    for example in split_data:
+        unique_id = generate_unique_random_number()
```
That is a good point. Originally I was keep the unique id's because the files kept getting overwritten. I have edited the script to generate Id's sequentially this should not be an issue anymore.

arinkulshi-skylight commented:
> Please add a section in the readme about running this (and maybe interpreting results?)
> 
> Additionally, do we need to update dependencies to include `datasets` I don't seem to have it installed inside my poetry shell.

I rewrote the script to auto populate all three datasets on one click. I also created a section in our readme pointing to the location of the data in gdrive. For this ticket I focused on only the data generation. Creating and interpreting results can be in the next ticket. I also added datasets to poetry. Completely forgot about that. Thanks for the feedback and help!

schreiaj commented on OCR/README.md:
```diff
@@ -7,77 +7,95 @@ pipx install poetry
+### Test Data Sets
+
+Here is the standarized form testing dataset
+https://drive.google.com/drive/folders/1WS2FYn0BTxWv0juh7lblzdMaFlI7zbDd
```
Nit: This is a public repo and that google drive shouldn't be public. I'd personally just remove this entirely and let folks run the script to download them.

arinkulshi-skylight replied:
I removed it. The script should do the data pull.

schreiaj reviewed (dismissed):
Minor nit but nothing blocking. LGTM
